docs/config/README.md

```diff
@@ -738,6 +738,7 @@ The module will be shown if any of the following conditions are met:
 | ------------ | --------------- | ------------------------------------- |
 | `style_root` | `"bold red"`    | The style used when the user is root. |
 | `style_user` | `"bold yellow"` | The style used for non-root users.    |
+| `show_always`| `false`         | Always shows the `username` module.   |
```

Comment:
Can we also add this to the list of conditions in the text above? (line 733). I'd add the suggestion there but GitHub won't let me for some reason.

Reply:
Done.